cockatrice/src/settings/gamefilterssettings.cpp

```diff
@@ -48,6 +48,17 @@ bool GameFiltersSettings::isShowPasswordProtectedGames()
     return previous == QVariant() ? true : previous.toBool();
 }
 
+void GameFiltersSettings::setHideIgnoredUserGames(bool hide)
+{
+    setValue(hide, "hide_ignored_user_games", "filter_games");
+}
+
+bool GameFiltersSettings::isHideIgnoredUserGames()
+{
+    QVariant previous = getValue("hide_ignored_user_games", "filter_games");
+    return previous == QVariant() ? false : previous.toBool();
```

Comment:
shouldn't we be using isNull() here and the rest of the uses?

Reply:
As far as I can tell, checking `isNull()` is not strictly equivalent to equality-checking invalid (i.e. unset) variants, as `QVariant` itself may be set with valid null values. I'm pretty new to Qt though, so may well have misinterpreted the docs here (https://doc.qt.io/qt-5/qvariant.html#QVariant).